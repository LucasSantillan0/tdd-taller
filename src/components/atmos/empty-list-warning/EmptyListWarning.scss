.empty-gif-list {
    display: flex;
    justify-content: center;
    align-items: center;
    gap: 1.5rem;
    padding: 2.5rem 0;

    @media (max-width: 23.5rem) {
        flex-direction: column;
        gap: .5rem

    }

    &__description {
        font-size: 1.5rem;
    }

}