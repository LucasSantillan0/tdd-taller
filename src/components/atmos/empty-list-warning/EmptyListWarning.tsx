import WarningIcon from "../../../assets/warning-icon.svg";

const EmptyListWarning = () => {
  return (
    <div className="empty-gif-list">
      <img className="empty-gif-list__icon" src={WarningIcon} alt="warning" />
      <p className="empty-gif-list__description">No posee gifs</p>
    </div>
  );
};

export default EmptyListWarning;
