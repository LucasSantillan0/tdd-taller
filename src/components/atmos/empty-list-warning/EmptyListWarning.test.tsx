import { render, screen } from "@testing-library/react";
import EmptyListWarning from "./EmptyListWarning";

describe("Empty list warning component", () => {
  test("Should render correctly", () => {
  });
});
