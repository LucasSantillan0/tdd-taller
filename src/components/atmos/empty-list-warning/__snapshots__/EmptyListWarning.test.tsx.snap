// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty list warning component Should render well 1`] = `
<DocumentFragment>
  <div
    class="empty-gif-list"
  >
    <img
      alt="warning"
      class="empty-gif-list__icon"
      src="warning-icon.svg"
    />
    <p
      class="empty-gif-list__description"
    >
      No posee gifs
    </p>
  </div>
</DocumentFragment>
`;
